pages/borrow/index.tsx

```diff
@@ -196,17 +201,27 @@ const Markets: NextPage = () => {
    * [2] reFetch function which can be called after deposit or withdraw and updates nft list
    */
   const [NFTs, isLoadingNfts, refetchNfts] = useFetchNFTByUser(wallet);
-
   const [isCreateMarketAreaOnHover, setIsCreateMarketAreaOnHover] =
     useState<boolean>(false);
 
   // fetches the sol price
   // TODO: create type for reserves and connection
-  async function fetchSolValue(reserves: any, connection: any) {
-    const slPrice = await fetchSolPrice(reserves, connection);
+  async function fetchSolValue(reserves: TReserve, connection: Connection) {
+    const slPrice = await fetchReservePrice(reserves, connection);
```

Comment:
rename constant to reserve price or something related to SPL tokens.
first it was SOL, now not anymore.

Reply:
done